src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -2484,6 +2491,92 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/calculateSize':
+    post:
+      x-alfresco-since: "5.2"
```

Comment:
Are we supporting this API from ACS 5.2 ?

Reply:
No, I have removed the same.